srcpkgs/TheForceEngine/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'TheForceEngine'
+pkgname=TheForceEngine
+version=1.09.410
+revision=1
+archs="x86_64"
```

Comment:
- builds fine on aarch
- this should be replaced with `case $XBPS_TARGET_MACHINE in somearch): broken="specific reason"`

Reply:
great, thank you for building that. I will find an example and update the template